examples/micro-http-server-ts/src/service.ts

```diff
@@ -0,0 +1,33 @@
+import * as qs from 'qs';
+import * as logger from './logger';
+
+import * as store from '@leapfrogtechnology/async-store';
+
+/**
+ * Set input params received from query in the store.
+ *
+ * @param {any} query
+ */
+export function storeParams(query) {
+  const { a, b } = qs.parse(query);
+
+  store.set({ a, b });
+
+  logger.debug(`Persisted a: ${a}`);
+  logger.debug(`Persisted b: ${b}`);
+}
+
+/**
+ * An example function making use of the request context
+ * set in the store asynchronously (with delay).
+ */
+export function doSomethingAsync() {
+  // Do something with the request with a delay.
+  logger.debug('Simulating delayed access');
+
+  setTimeout(() => {
+    const [a, b, sum] = [store.get('a'), store.get('b'), store.get('sum')];
```

Comment:
```suggestion
	const res = store.getAll();
```

Reply:
We have `getAll` function to fetch all the values of the store.

Reply:
`store.getAll()` returns `TypeError: store.getAll is not a function`

Reply:
looks like we've not released it yet.

Reply:
@cham11ng We can release it, if it's not released yet.